app/src/main/java/com/marverenic/music/player/ScrobblerExtension.java

```diff
@@ -0,0 +1,34 @@
+package com.marverenic.music.player;
```

Comment:
I'm wondering if this should go in a separate package along with `SlsMessenger`, or if there should be an `com.marverenic.music.player.extensions` package. Thoughts?